Makefile

```diff
@@ -1708,6 +1710,80 @@ frwiki_tuning_reports: \
 	tuning_reports/frwiki.goodfaith.md
 
 
+############################# Galician Wikipedia ################################
+datasets/glwiki.human_labeled_revisions.60k_2019.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/glwiki/85/ > $@
```

Comment:
This block shouldn't be here.  Was it auto-generated?

Reply:
I added this manually, see comment below.